core/src/main/java/org/apache/accumulo/core/compaction/CompactionSettings.java

```diff
@@ -21,6 +21,7 @@
 
 public enum CompactionSettings {
 
+  SF_NO_SAMPLE(new NullType()),
```

Comment:
Could have static instances of these, but I don't think it's necessary since they're wrapped in the enum (e.g. we'd only have one instance anyways). Is that right?